src/js/photo.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+function addPhoto() {

```

Comment:
```suggestion
const addPhoto = () => {
```

Yo creo que es mejor poner un arrow ya que no solemos usar funciones normales.